### ISSUE TITLE
Increased glutamic acid in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6039]
**Preferred term label:** Increased glutamic acid in urine

**Synonyms** Hyperglutamic aciduria

**Definition (free text, please give PubMed ID)** Concentration of glutamic acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Aminoaciduria HP:0003355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
glutamate is the singly-negative anion  form of glutamic acid, and glutamate is the form found in body fluids, so this is actually a duplicate with #6036
